client/jetpack-cloud/components/guided-tour/index.tsx

```diff
@@ -10,16 +10,39 @@ import { preferencesLastFetchedTimestamp } from 'calypso/state/preferences/selec
 import './style.scss';
 
 export interface Tour {
-	targetId: string;
+	target: string;
```

Comment:
I have updated this to have a more generic selector instead of limiting the component to select elements only by ID. This is also necessary as sometimes we target elements that are children of a part we do not control.